src/Simulation/CsharpGeneration.Tests/Circuits/UnitTests.qs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Core {
+    @Attribute ()
+    newtype Attribute = Unit;
+
+    @Attribute ()
+    newtype TestOperation = String;
```

Comment:
May I suggest that this attribute live in M.Q.Diagnostics?

Reply:
This attribute normally lives in the core library. Since we don't have access to core here, this is just trying to replicate that behavior.

Reply:
@cgranade 
Yes, I think having it live in the Diagnostics namespace makes sense, too. I wasn't sure whether we would want it auto-opened so I added it to Core, but I am happy either way. We should consider having a file template for unit tests that contains that namespace an a stump to make it easier to get going.